src/game/round.ts

```diff
@@ -30,7 +30,7 @@ export abstract class Round {
         return out;
     }
 
-    public isTimedRound(): boolean {
-        return true;
+    public isTimedRound(diff: GameDifficulty): boolean {
+        return diff === GameDifficulty.Noob ? false : true;
```

Comment:
Just a comment here @lramo062, `===` returns a boolean, so you don't need to do it manually.

This is how it would normally be done:

```
return diff !== GameDifficulty.Noob;
```

Reply:
😅 yeah good point appreciate the comments!

Reply:
No problem at all, I'm here to help!

Reply:
sorry I didn't do a full review ;), good catch though brandon.